src/modules/Builder/components/Configuration/components/Wireless/Wireless.jsx

```diff
@@ -0,0 +1,154 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+import Card from "common/containers/Card";
+import Input from "common/components/Input";
+import InputGroup from "common/components/InputGroup";
+
+import * as RPC from "api/rpc/blackmagic";
+
+export default class Wireless extends Component {
+    static WarningMessege() {
+        return (
+            <div className="text-warning">
+                Пароль должен быть пустой либо содержать от 8 до 63 символов,
+                иначе сборка завершится неудачно.
+            </div>
+        );
+    }
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            PSK: "",
+            SSID: "",
+            disabledState: true,
+            warningStatus: false,
+            enableWireless: true,
+            previosPSK: "",
+        };
+
+        this.onCheckboxChange = this.onCheckboxChange.bind(this);
+        this.onChangeData = this.onChangeData.bind(this);
+        this.PSKValidor = this.PSKValidor.bind(this);
+    }
+
+    componentDidMount() {
+        RPC.fetchDefaultConfigurationParams()
+            .then((defaultConfig) => {
+                this.setState(() => ({ SSID: defaultConfig.SSID }));
+            });
+    }
+
+    onCheckboxChange(value) {
```

Comment:
* You can use one `setState`;
* `disabledState` duplicates the assignment of `enableWireless`, you can get rid of it.